### PR TITLE
v3.0/reference, v2.1/reference, dev/reference: update stats related configuration

### PR DESCRIPTION
**Pull request by lamxTyler**
### What is changed, added or deleted? 

Configuration of statistics is updated.

### What is the related PR or file link(s)? 

- master: https://github.com/pingcap/tidb/pull/10771
- 3.0: https://github.com/pingcap/tidb/pull/10811
- 2.1: https://github.com/pingcap/tidb/pull/10772

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 1]
```diff
@@ -172,12 +172,15 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ TiDB 重载统计信息，更新表行数，检查是否需要自动 analyze，利用 feedback 更新统计信息以及加载列的统计信息的时间间隔。
 + 默认：3s
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新统计信息
++ 当 `stats-lease` 为 0 时， TiDB 会以 3s 的时间间隔去读取最新的统计信息到内存中，但和统计信息更新相关的操作，例如维护总行数和修改行数，自动 analyze 以及 feedback 等操作都不会再进行。
```
Comment (zz-jason) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```
- 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
	- `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
	- `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze，主动更新统计信息
	- `mysql.stats_feedback`：TiDB 不再根据被查询的数据反馈的部分统计信息更新表和索引的统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 2]
```diff
@@ -172,12 +172,21 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新统计信息
++ 当 `stats-lease` 为 0 时， 上述所有操作都不会再进行。
+
+### `enable-update-stats`
+
```
Comment (zz-jason) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```
> 注：该配置项由 2.1.13 版本中引入，仅在 TiDB 版本大于等于 v2.1.13 的 2.1 版本中支持。
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 3]
```diff
@@ -172,12 +172,21 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ TiDB 是否做统计信息更新相关操作，例如维护总行数和修改行数，自动 analyze 以及 feedback 等。
++ 默认： true
++ 由于统计信息更新操作的周期由 `stats-lease` 决定，因此该参数只在 `stats-lease` 不为 0 的情况下生效。
```
Comment (zz-jason) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```
当 enable-update-stats 为 false 时，TiDB 不再自动修改这些表：
	- `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
	- `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze，主动更新统计信息
	- `mysql.stats_feedback`：TiDB 不再根据被查询的数据反馈的部分统计信息更新表和索引的统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 4]
```diff
@@ -172,12 +172,26 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新统计信息
++ 当 `stats-lease` 为 0 时， 上述所有操作都不会再进行。
+
+### `enable-update-stats`
+
+> 注：该配置项由 2.1.13 版本中引入，仅在 TiDB 版本大于等于 v2.1.13 的 2.1 版本中支持。
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
> **注意：**
>
> 该配置项由 2.1.13 版本中引入，仅在 TiDB 版本大于等于 v2.1.13 的 2.1 版本中支持。
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 5]
```diff
@@ -172,12 +172,26 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+
++ TiDB 是否做统计信息更新相关操作，例如维护总行数和修改行数，自动 analyze 以及 feedback 等。
++ 默认： true
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
+ 默认：true
```

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" extension in your VS Code editor, so it will check all potential issues that fail the CI check. 

@lamxTyler If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

Ref: https://github.com/pingcap/docs-cn/pull/1494

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 6]
```diff
@@ -224,12 +224,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ TiDB 重载统计信息，更新表行数，检查是否需要自动 analyze，利用 feedback 更新统计信息以及加载列的统计信息的时间间隔。
 + 默认值：3s
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化更新到系统表中
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新内存中缓存的统计信息
++ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
+ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息并更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
```

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 7]
```diff
@@ -224,12 +224,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze，主动更新统计信息
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze 和主动更新统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 8]
```diff
@@ -172,12 +174,28 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+
+### `enable-update-stats`
+
+> **注意：**
+>
+> 该配置项由 2.1.13 版本中引入，仅在 TiDB 版本大于等于 v2.1.13 的 2.1 版本中支持。
+
++ TiDB 是否做统计信息更新相关操作，例如维护总行数和修改行数，自动 analyze 以及 feedback 等。
++ 默认：true
++ 由于统计信息更新操作的周期由 `stats-lease` 决定，因此该参数只在 `stats-lease` 不为 0 的情况下生效。当 enable-update-stats 为 false 时，TiDB 不再自动修改这些表：
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
+ 由于统计信息更新操作的周期由 `stats-lease` 决定，因此该参数只在 `stats-lease` 不为 0 的情况下生效。当 `enable-update-stats` 为 false 时，TiDB 不再自动修改这些表：
```

File: v3.0/reference/configuration/tidb-server/configuration-file.md [hunk 9]
```diff
@@ -225,12 +225,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化更新到系统表中
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新内存中缓存的统计信息
++ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze，主动更新统计信息
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze 和主动更新统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 10]
```diff
@@ -172,12 +174,28 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms` 和 `mysql.stats_buckets`：TiDB 不再自动 Analyze，主动更新统计信息
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms` 和 `mysql.stats_buckets`：TiDB 不再自动 Analyze 和主动更新统计信息
```

File: v3.0/reference/configuration/tidb-server/configuration-file.md [hunk 11]
```diff
@@ -225,12 +225,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ TiDB 重载统计信息，更新表行数，检查是否需要自动 analyze，利用 feedback 更新统计信息以及加载列的统计信息的时间间隔。
 + 默认值：3s
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化更新到系统表中
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新内存中缓存的统计信息
++ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
+ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息并更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
```

File: v3.0/reference/configuration/tidb-server/configuration-file.md (hunk 9 above)
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze 和主动更新统计信息
```

File: v3.0/reference/configuration/tidb-server/configuration-file.md [hunk 12]
```diff
@@ -225,12 +225,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze，主动更新统计信息
+    - `mysql.stats_feedback`：TiDB 不再根据被查询的数据反馈的部分统计信息更新表和索引的统计信息analyze 以及 feedback 等操作都不会再进行。
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_feedback`：TiDB 不再根据被查询的数据反馈的部分统计信息更新表和索引的统计信息，analyze 以及 feedback 等操作都不会再进行。
```

File: v3.0/reference/configuration/tidb-server/configuration-file.md [hunk 13]
```diff
@@ -225,12 +225,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化更新到系统表中
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新内存中缓存的统计信息
++ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息并更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze 和主动更新统计信息
```
Comment (yikeke) [inline v3.0/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 analyze 和主动更新统计信息
```
统一使用 `analyze` 或 `Analyze`。不要混用。

File: dev/reference/configuration/tidb-server/configuration-file.md [hunk 14]
```diff
@@ -224,12 +224,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
     - 每隔 `stats-lease` 时间，TiDB 会检查统计信息是否有更新，如果有会将其更新到内存中
-    - 每隔 `5 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化持久化下来
+    - 每隔 `20 * stats-lease` 时间，TiDB 会将 DML 产生的总行数以及修改的行数变化更新到系统表中
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有表或者索引需要自动 analyze
     - 每隔 `stats-lease` 时间，TiDB 会检查是否有列的统计信息需要被加载到内存中
+    - 每隔 `200 * stats-lease` 时间，TiDB 会将内存中缓存的 feedback 写入系统表中
+    - 每隔 `5 * stats-lease` 时间，TiDB 会读取系统表中的 feedback，更新内存中缓存的统计信息
++ 当 `stats-lease` 为 0 时，TiDB 会以 3s 的时间间隔周期性的读取系统表中的统计信息并更新内存中缓存的统计信息。但不会自动修改统计信息相关系统表，具体来说，TiDB 不再自动修改这些表：
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 Analyze 和主动更新统计信息
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms`/`mysql.stats_buckets` 和 `mysql.stats_top_n`：TiDB 不再自动 analyze 和主动更新统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 15]
```diff
@@ -174,12 +176,28 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+
+> **注意：**
+>
+> 该配置项由 2.1.13 版本中引入，仅在 TiDB 版本大于等于 v2.1.13 的 2.1 版本中支持。
+
++ TiDB 是否做统计信息更新相关操作，例如维护总行数和修改行数，自动 analyze 以及 feedback 等。
++ 默认：true
++ 由于统计信息更新操作的周期由 `stats-lease` 决定，因此该参数只在 `stats-lease` 不为 0 的情况下生效。当 `enable-update-stats` 为 false 时，TiDB 不再自动修改这些表：
+    - `mysql.stats_meta`：TiDB 不再自动记录事务中对某张表的修改行数，也不会更新到这个系统表中
+    - `mysql.stats_histograms` 和 `mysql.stats_buckets`：TiDB 不再自动 Analyze 和主动更新统计信息
```
Comment (yikeke) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]:
```suggestion
    - `mysql.stats_histograms` 和 `mysql.stats_buckets`：TiDB 不再自动 analyze 和主动更新统计信息
```

File: v2.1/reference/configuration/tidb-server/configuration-file.md [hunk 16]
```diff
@@ -3,6 +3,8 @@ title: TiDB 配置文件描述
 category: reference
 ---
 
+<!-- markdownlint-disable MD001 -->
+
 # TiDB 配置文件描述
 
 <!-- markdownlint-disable MD001 -->
```
Comment (winoros) [inline v2.1/reference/configuration/tidb-server/configuration-file.md]: We need to keep this line together with line#6?

Comment (yikeke) [reply v2.1/reference/configuration/tidb-server/configuration-file.md]: > We need to keep this line together with line#6?

No, it's duplicating the line 10. We should remove the line 10.

**Pull request merged**